CHANGELOG.md

```diff
@@ -3,6 +3,22 @@ aws-parallelcluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the AWS ParallelCluster cookbook.
 
+2.6.2
+-----
+
+**ENHANCEMENTS**
+
+**CHANGES**
+- Upgrade EFA installer to version 1.8.4
+  - Kernel module: efa-1.5.1 (no change)
+  - RDMA core: rdma-core-25.0 (distributed only) (no change)
+  - Libfabric: libfabric-aws-1.9.0amzn1.1 (no change)
+  - Open MPI: openmpi40-aws-4.0.3 (vs. openmpi40-aws-4.0.2)
```

Comment:
minor: let's use the same convention we have in previous changes: (updated from .. in place of "vs.")

Reply:
done